### DISCUSSION TITLE
Original Yolov5 source?

### DISCUSSION THREAD

**Discussion by lucysumi** [ultralytics/yolov5#9120] [Q&A]
There has been so many updates on Yolov5. Can I please get the source to original Yolov5 architecture (Diagram)?
And if I want to cite Yolov5, how should I ? Should I cite the updated version or the original one? This is very confusing.

**Comment by spacewalk01** [comment]
This is the architecture of YOLOv5:
Diagram: https://user-images.githubusercontent.com/31005897/172404576-c260dcf9-76bb-4bc8-b6a9-f2d987792583.png
Explanation: https://github.com/ultralytics/yolov5/issues/6998#issue-1170533269

I think you can cite the recent version of yolov5 as far as I know it didn't change much

**Reply by lucysumi** [reply]
Thanks alot @spacewalk01
